python/segyio/segy.py

```diff
@@ -788,6 +810,12 @@ def __setitem__(inner, index, val):
 
                 _segyio.write_textheader(self.xfd, index, val)
 
+            def __repr__(inner):
+                return "Text(external_headers = {})".format(self.ext_headers)
+
+            def __str__(inner):
```

Comment:
Probably the right thing to do. But perhaps there are other strong opinions on this one.